nilearn/maskers/surface_labels_masker.py

```diff
@@ -352,14 +352,27 @@ def fit_transform(self, img, y=None):
             This parameter is unused.
             It is solely included for scikit-learn compatibility.
 
+        confounds : :class:`numpy.ndarray`, :obj:`str`,\
+                    :class:`pathlib.Path`, \
+                    :class:`pandas.DataFrame` \
+                    or :obj:`list` of confounds timeseries, default=None
+            Confounds to pass to :func:`nilearn.signal.clean`.
```

Comment:
I am going to be annoying (and a hypocrite because I did not do it myself) but do you think you could parametrize the test with confounds to check all types we support in the doc string?

Reply:
done